tests/factories.py

```diff
@@ -0,0 +1,14 @@
+import factory.fuzzy
+from subjects import models
+
+
+class SubjectFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = models.Subject
+
+    name = factory.fuzzy.FuzzyText()
+    parent = factory.fuzzy.FuzzyInteger(0)
```

Comment:
I don't think the parent should be a random integer, that is important to the hierarchical structure of the subjects. We should probably set it to 0 or 1 and then manually adjust the parent in the tests as needed.

Reply:
I'm thinking `parent` reefers to id of the parent subject object. So, I'm not sure if I should set it to 0 or 1 here.